### PR TITLE
docs: generate spa for now 2.0

### PR DESCRIPTION
**Pull request by MajesticPotatoe**
## Description
SPA version of docs

## Motivation and Context
Potential route for now 2.0

## How Has This Been Tested?
 visually

## Markup:
-NA-

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/package.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "author": "John Leider <john@vuetifyjs.com>",
   "version": "2.2.12",
   "scripts": {
-    "dev": "cross-env node --max_old_space_size=4096 server",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server --config build/webpack.config.js --hot --progress",
     "start": "cross-env NODE_ENV=production node server",
-    "build": "rimraf dist && cross-env NODE_ENV=production webpack --config build/webpack.prod.config.js --no-progress --hide-modules",
+    "build": "rimraf dist && cross-env NODE_ENV=production webpack --config build/webpack.config.js --no-progress --hide-modules",
```
Comment (styfle) [inline packages/docs/package.json]: You will need to configure the correct output directory in the [project dashboard](https://zeit.co/blog/advanced-project-settings)  or you can add `mv packages/docs/dist public` to move the build output into the `public` directory.

File: now.json [hunk 2]
```diff
@@ -1,10 +1,12 @@
 {
+  "version": 2,
   "name": "vuetifyjs.com",
   "alias": "vuetifyjs.com",
```
Comment (styfle) [inline now.json]: Remove `name` and `alias` here. You can add the domain in the project settings.

**Comment by styfle** [comment]
I just tested this PR by assigning the the Output Directory to `packages/docs/dist` in the [Project settings](https://zeit.co/blog/advanced-project-settings) and it works!

I also needed to remove the `name` and `alias` properties from `now.json` since the domain belongs in the Project settings well.

See a demo here: `https://vuetify-spa-ani0dzilv.now.sh`.

**Comment by johnleider** [comment]
I can't get this to run.

**Comment by MajesticPotatoe** [comment]
got this working with `now` command using the cli
had to add a `.nowignore` to ignore some of the non-needed file directories otherwise it would yell about file limits locally, however this wouldn't really be the case if using GH integration. 
seems to also work if you just push the build files directory

**Comment by MajesticPotatoe** [comment]
https://vuetifyjscom-9xh2a26gi.now.sh/

**Comment by MajesticPotatoe** [comment]
@KaelWD any insight on getting hot-reload/dev working on this? currently it builds, but errors out console side.

**Comment by MajesticPotatoe** [comment]
being closed in favor of #10751

**Pull request closed without merging**